ocw/lib/cleanup.py

```diff
@@ -47,6 +47,23 @@ def list_clusters():
                     type(e).__name__, vault_namespace), traceback.format_exc())
 
 
+def delete_snapshots():
+    cfg = ConfigFile()
+    if cfg.has('snapshots'):
+        for vault_namespace in cfg.getList(['snapshots', 'namespaces'], ['']):
+            try:
+                delete_after_days = date.today() - timedelta(days=cfg.getInt(['snapshots', 'delete_after_days'], 30))
+                logger.info("[{}] Scanning for too old snapshots. Everything older than {} will be deleted".format(
+                    vault_namespace, delete_after_days))
+                deleted_snapshots = EC2(vault_namespace).delete_snapshots_older_than(delete_after_days)
+                send_snapshots_notification(vault_namespace, deleted_snapshots)
+            except Exception as e:
+                logger.exception("[{}] Cleanup snapshots failed!".format(vault_namespace))
+                send_mail('{} on cleanup snapshots in [{}]'.format(
+                    type(e).__name__, vault_namespace), traceback.format_exc())
+
+
 def init_cron():
     getScheduler().add_job(cleanup_run, trigger='interval', minutes=60, id='cleanup_all')
     getScheduler().add_job(list_clusters, trigger='interval', hours=18, id='list_clusters')
+    getScheduler().add_job(delete_snapshots, trigger='interval', hours=18, id='delete_snapshots')
```

Comment:
I'm wonder, why you need that extra scheduler? I think it is just a cleanup task

Reply:
can I add several functions in same scheduler job ? don't think it would be right to put this two different logics into same function . also this gives me more flexibility on different intervals for different cleaners

Reply:
I see it differently. We have a `XXX.cleanup_all()` for all providers. And that function handle all cleanup jobs for orphan artifacts in each provider. 
These `snapshots` are ec2 related and with this only valid for ec2.